torch/_prims/nvfuser_executor.py

```diff
@@ -41,7 +42,10 @@ def to_nvfuser_template_args(args):
     def to_nvfuser(arg):
         if isinstance(arg, torch.Tensor):
             return nvFuserTensorTemplate(
-                arg.size(), arg.stride(), getnvFuserDtype(arg.dtype)
+                arg.size(),
+                arg.stride(),
+                getnvFuserDtype(arg.dtype),
+                arg.device.type == "cpu",
```

Comment:
```suggestion
                arg.is_cpu,
```

Reply:
errr.. that `is_cpu` was what I had in the first place and it triggers a linter error. https://github.com/pytorch/pytorch/runs/8161952307?check_suite_focus=true

Reply:
Skip this line for linter? 😄